modules/serverless-enabling-tls-internal-traffic.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * /serverless/security/serverless-config-tls.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-enabling-tls-internal-traffic_{context}"]
+= Enabling TLS authentication for internal traffic
+
+{ServerlessProductName} supports TLS edge termination by default, so HTTPS traffic from end users is encrypted. However, internal traffic behind the OpenShift route is forwarded to applications by using plain data. By enabling TLS for internal traffic, the traffic sent between components is encrypted, which makes this traffic more secure.
+
+:FeatureName: Internal TLS encryption support
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have installed the OpenShift (`oc`) CLI.
+* If you want to enable internal TLS with a Service Mesh integration, you must enable Service Mesh with mTLS.
```

Comment:
@nak3 Does this integration with Service Mesh work? I haven't seen this setup anywhere. Please see thread
https://coreos.slack.com/archives/CF5ANN61F/p1663599018458669

Reply:
Maybe this sentence means: "When SM is enabled with mTLS you get internal encryption automatically and then the rest of this section is irrelevant?

Reply:
Yes, that's right.

This sentence should not be in the Prerequisites but note section or something. I wanted to explain it as:

> If you want to enable internal TLS with a Service Mesh integration, you must enable Service Mesh with mTLS by following the section - [Enabling Knative Serving metrics when using Service Mesh with mTLS](https://docs.openshift.com/container-platform/4.11/serverless/admin_guide/serverless-ossm-setup.html#serverless-ossm-enabling-serving-metrics_serverless-ossm-setup) instead.

Reply:
@nak3 Thanks! The suggested paragraph makes it very clear.